### PR TITLE
Move Cache to src

### PR DESCRIPTION
**Pull request by zeroadam**
Moved old cache to Friendica\Core namespace.

Related to Issue #3878

### CODE REVIEW

**Comment by MrPetovan** [comment]
So, the deal with the `require_once` within the code was to emulate lazy-loading (now provided by the autoloader for classes in `src`). They shouldn't even be there in the first place, but we still need them as `require_once` because it would cause a `symbol not found` fatal error after the `include file not found` warning. I'd rather have a `include file not found` fatal error right away until we can properly strip those statements out.

File: include/identity.php [hunk 1]
```diff
@@ -92,30 +95,35 @@ function profile_load(App $a, $nickname, $profile = 0, $profiledata = array()) {
 
 	$theme_info_file = "view/theme/" . current_theme() . "/theme.php";
 	if (file_exists($theme_info_file)) {
-		require_once $theme_info_file;
+		include_once $theme_info_file;
```
Comment (MrPetovan) [inline include/identity.php]: Please revert this change.

File: include/identity.php [hunk 2]
```diff
@@ -536,36 +572,38 @@ function get_birthdays() {
...
+	);
 }
 
 
-function get_events() {
-
-	require_once 'include/bbcode.php';
+function get_events()
+{
+	include_once 'include/bbcode.php';
```
Comment (MrPetovan) [inline include/identity.php]: Please revert this line.

File: include/queue.php [hunk 3]
```diff
@@ -64,18 +68,21 @@ function queue_run(&$argv, &$argc) {
 
 	// delivering
 
-	require_once 'include/salmon.php';
+	include_once 'include/salmon.php';
```
Comment (MrPetovan) [inline include/queue.php]: Please revert this line.

File: include/identity.php (hunk 1 above)
Comment (zeroadam) [reply include/identity.php]: Done

File: include/identity.php (hunk 2 above)
Comment (zeroadam) [reply include/identity.php]: Done

File: include/queue.php (hunk 3 above)
Comment (zeroadam) [reply include/queue.php]: Done

File: include/identity.php [hunk 4]
```diff
@@ -452,33 +480,36 @@ function profile_sidebar($profile, $block = 0) {
...
+	if (! local_user() || $a->is_mobile || $a->is_tablet) {
 		return $o;
+	}
 
-//		$mobile_detect = new Mobile_Detect();
-//		$is_mobile = $mobile_detect->isMobile() || $mobile_detect->isTablet();
-
-//		if ($is_mobile)
-//			return $o;
+	/**
```
Comment (rabuzarus) [inline include/identity.php]: you need have to look that comments don't start with `/**` because all stuff with `/**` will be added to the doxygen documentation.
Alternatives: `/*` or `/* *`

Comment (MrPetovan) [reply include/identity.php]: Good catch.
